packages/components/config-provider/src/config-provider.ts

```diff
@@ -30,6 +37,7 @@ export default defineComponent({
 
   setup(props, { slots }) {
     provideLocale()
+    Object.assign(messageConfig, props.message || {})
```

Comment:
I think you should make sure that this value is reactive, for example:

```ts
watch(() => props.message, (val) => {
  Object.assign(messageConfig, val ?? {})
}, { immediate: true })
```

Reply:
Then you should add test against updating the message maximum value.

Reply:
You are right.